src/nupic/algorithms/SpatialPooler.cpp

```diff
@@ -574,7 +575,7 @@ void SpatialPooler::initialize(vector<UInt> inputDimensions,
 
   tieBreaker_.resize(numColumns_);
   for (UInt i = 0; i < numColumns_; i++) {
-    tieBreaker_[i] = 0.01 * rng_.getReal64();
+    tieBreaker_[i] = pow(10, -1*(getPrecision()+1)) * rng_.getReal64(); // (PRECISION+1)-th place
```

Comment:
I'm getting `inf` for these tie breaker values as opposed to random values within the same precision.  Is that intended?